pkg/monitortests/node/watchnodes/node.go

```diff
@@ -73,8 +80,16 @@ func startNodeMonitoring(ctx context.Context, m monitorapi.RecorderWriter, clien
 						WithAnnotation(monitorapi.AnnotationRoles, nodeRoles(node)).
 						HumanMessage("node is ready")).Build(now, now),
 			}
-		}
 
+		case isNodeNotReady && !didConfigChange:
```

Comment:
This case will never be reached because every time `isNodeNotReady` is true, one of the branches above will be taken.

Reply:
Ah. that is what I get for trying to be cute.

I'm just going to add a new function for the unexpected section. I think that may be more clear and less error prone. I wanted to try and reuse the ready check but I think I am overthinking/over optimizing this.

Reply:
Updated. PTAL.